drake/examples/Pendulum/Pendulum.h

```diff
@@ -0,0 +1,41 @@
+#ifndef _PENDULUM_H_
+#define _PENDULUM_H_
+
+#include <iostream>
+#include "DrakeSystem.h"
+
+using namespace std;
+using namespace Eigen;
```

Comment:
as I rule I try to avoid it, but Pendulum.h feels like it is at the bottom of the include tree -- it is only included by a bunch of short apps (runDynamics.cpp runEnergyShaping.cpp etc).  Having an entire Pendulum.cpp and setting up the linking felt like overkill for this.